.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: java
+jdk:
+  - oraclejdk8
+  - oraclejdk7
+branches:
+  only:
+    - master
+install: "mvn validate -DskipTests=true -Dcamus.version=0.0.0"
+script: "mvn clean test -Dcamus.version=0.0.0 -B"
+before_install:
+  - sed -i.bak -e 's|https://nexus.codehaus.org/snapshots/|https://oss.sonatype.org/content/repositories/codehaus-snapshots/|g' ~/.m2/settings.xml
```

Comment:
It's [one of the pitalls](https://github.com/vinted/camus/pull/31#issuecomment-236493589): a Travis issues that for some reason [no one bothers to fix](https://github.com/travis-ci/travis-ci/issues/4629).

Otherwise had to use custom `install`and `script` due to use of `camus.version` - default  versions do not work.

Also kad to expicitly call `validate` step on install because `maven-install-plugin` installs local dependencies [on validate](https://github.com/vinted/camus/blob/ddd0d4b1141d9b695b8878570d49bf8fe7c08dee/pom.xml#L203)]